fastseq/models/unilm_hf/modeling_unilm.py

```diff
@@ -761,7 +761,7 @@ def generate(self, input_ids, attention_mask, decoder_start_token_id,
             num_beams: beam size for beam search
             num_return_sequences: num for return sequence for beam search
         """
-        max_seq_length = kwargs.pop("max_gen_length", 48)
+        max_seq_length = kwargs.pop("max_length", 48)
```

Comment:
Does [here](https://github.com/microsoft/fastseq/tree/main/examples/unilm#setting) also need to be updated?

Reply:
No need to update it. The parameter for fastseq-cli is `max_gen_length`. [Here](https://github.com/microsoft/fastseq/blob/main/fastseq_cli/transformers_generate.py#L290)